### PR TITLE
Search result list gets sorted by clikcing on the title of the column.

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose

Issue: https://issues.folio.org/browse/UIMARCAUTH-135
The search result list doesn't sort by clicking on the title of column

## Screenshots

chrome_EgtxPL3gEt

### CODE REVIEW

File: src/hooks/useSortColumnManager/useSortColumnManager.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const useSortColumnManager = (options) => {
 
     let newOrder;
 
-    if (options?.sortableColumns && !options?.sortableColumns.includes(name)) return;
+    if (options?.sortableColumns && !Object.values(options.sortableColumns).includes(name)) return;
```
Comment (BogdanDenis) [inline src/hooks/useSortColumnManager/useSortColumnManager.js:20]: In tests we're passing `sortableColumns` as an array, but `sortableSearchResultListColumns` is an object.
I'd prefer this to be an array so we don't have to use `Object.values`.
And after changing it to array we also should update how it's used in `AuthoritiesSearch`

File: src/routes/SearchRoute/SearchRoute.js [hunk 2]
```diff
@@ -34,12 +34,17 @@ const SearchRoute = ({ children }) => {
...
+    searchResultListColumns.HEADING_REF,
+    searchResultListColumns.HEADING_TYPE,
+  ];
   const {
     sortOrder,
     sortedColumn,
     onChangeSortOption,
     onHeaderClick,
-  } = useSortColumnManager({ sortableColumns: sortableSearchResultListColumns });
+  } = useSortColumnManager(sortableColumns);
```
Comment (BogdanDenis) [inline src/routes/SearchRoute/SearchRoute.js:47]: Should be an object with `sortableColumns` properties here

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/routes/SearchRoute/SearchRoute.js (hunk 2 above)
Comment (Terala-Priyanka) [reply src/routes/SearchRoute/SearchRoute.js:47]: as per our discussion, shouldn't it be an array?

Comment (Terala-Priyanka) [reply src/routes/SearchRoute/SearchRoute.js]: passed sortableColumns as an object parameter

**Review by BogdanDenis — APPROVED** [review]

**Review by ODanyor — APPROVED** [review]

**Pull request merged**